test/conformance/kubetestgen/README.md

```diff
@@ -0,0 +1,19 @@
+# Kubetestgen
+
+kubetestgen generates a list of behaviors for a resource based on the OpenAPI schema.
+
+Example usage for PodSpec:
+
+```
+bazel build //test/conformance/kubetestgen:kubetestgen
```

Comment:
it should bulid with go build as well, right? you may want to show that as well.

Reply:
It could be built with go build but bazel works as long as we explicitly call the executable via `./bazel-out/.../kubetestgen`